### PR TITLE
Feature/gh 810 review recovery

### PR DESCRIPTION
**Pull request by DmitryBespalov**
- New RecoverRecoveryFee screen
- Renamings and movings of files
- Changes to the PaymentMethod controllers
- New transaction `estimations` endpoint implementation (currently falls back to normal estimation)

### CODE REVIEW

File: MultisigWallet/MultisigWalletDomainModel/Wallet/DeploymentDomainService.swift [hunk 1]
```diff
@@ -88,7 +88,7 @@ public class DeploymentDomainService {
             let request = SafeCreationRequest(saltNonce: DomainRegistry.encryptionService.randomSaltNonce(),
                                               owners: owners,
                                               confirmationCount: wallet.confirmationCount,
-                                              paymentToken: Token.Ether.address)
+                                              paymentToken: wallet.feePaymentTokenAddress ?? Token.Ether.address)
```
Comment (sche) [inline MultisigWallet/MultisigWalletDomainModel/Wallet/DeploymentDomainService.swift:91]: how did it work before? :)

File: SafeUIKit/SafeAppUI/MainFlow/OnboardingFlow/SetupSafeFlow/RecoverSafe/RecoverRecoveryFeeViewController.swift [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+    var isFinished: Bool = false
+
+    enum Strings {
+
+        static let title = LocalizedString("recover_safe_title", comment: "Create Safe")
+        static let subtitle = LocalizedString("insufficient_funds", comment: "Insufficient funds header.")
+        static let subtitleDetail = LocalizedString("recovering_requires_fee", comment: "Explanation text.")
+        static let amountReceived = LocalizedString("safe_balance", comment: "Safe balance")
+    }
+
```
Comment (sche) [inline SafeUIKit/SafeAppUI/MainFlow/OnboardingFlow/SetupSafeFlow/RecoverSafe/RecoverRecoveryFeeViewController.swift:35]: extra space

**Review by sche — APPROVED** [review]
👍

File: MultisigWallet/MultisigWalletDomainModel/Wallet/DeploymentDomainService.swift (hunk 1 above)
Comment (DmitryBespalov) [reply MultisigWallet/MultisigWalletDomainModel/Wallet/DeploymentDomainService.swift:91]: ✨✨✨✨✨✨✨✨
✨✨✨magic✨✨✨
✨✨✨✨✨✨✨✨

**Pull request merged**